tf5muxserver/mux_server_GetProviderSchema.go

```diff
@@ -14,7 +14,7 @@ import (
 
 // GetProviderSchema merges the schemas returned by the
 // tfprotov5.ProviderServers associated with muxServer into a single schema.
-// Resources, data sources, ephemeral resources, and functions must be returned
+// Resources, data sources, ephemeral resources, list resources, and functions must be returned
 // from only one server. Provider and ProviderMeta schemas must be identical between all servers.
 func (s *muxServer) GetProviderSchema(ctx context.Context, req *tfprotov5.GetProviderSchemaRequest) (*tfprotov5.GetProviderSchemaResponse, error) {
```

Comment:
Shall we also update the test in `tf5muxserver/mux_server_GetProviderSchema_test.go`? & similarly for v6.